### PR TITLE
Add timestamp log to application

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -523,7 +523,7 @@
     t.index ["script_level_id"], name: "index_levels_script_levels_on_script_level_id", using: :btree
   end
 
-  create_table "libraries", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "libraries", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```
Comment (clareconstantine) [inline dashboard/db/schema.rb:526]: is this `mb4` part intentional?

File: dashboard/db/schema.rb [hunk 2]
```diff
@@ -1066,7 +1067,7 @@
     t.index ["user_id", "script_id", "level_id"], name: "index_puzzle_ratings_on_user_id_and_script_id_and_level_id", unique: true, using: :btree
   end
 
-  create_table "queued_account_purges", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "queued_account_purges", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```
Comment (clareconstantine) [inline dashboard/db/schema.rb:1070]: same question - did you mean to add `mb4`?

**Review by clareconstantine — APPROVED** [review]

File: dashboard/db/schema.rb (hunk 1 above)
Comment (mehalshah) [reply dashboard/db/schema.rb:526]: No - need to remove this

File: dashboard/db/schema.rb (hunk 2 above)
Comment (mehalshah) [reply dashboard/db/schema.rb:1070]: Same, need to remoe this

**Pull request merged**